javascript/javascript1/week2/smart-ease2.js

```diff
@@ -0,0 +1,135 @@
+//Flight booking fullname function
+
+function getFullname (firstname, surname, useFormalName) {
+    if (useFormalName == true) {
+        return "Lord " + firstname + " " + surname;
+    }
+    else {
+        return firstname + " " + surname;
+    }
+}
+
+//two variables assigned to getFullname function.
+let fullname1 = getFullname("Math", "Anton", true);
+let fullname2= getFullname("Mia", "David");
```

Comment:
Both are const. Why let? Good reading https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/

Reply:
Try also `getFullname();` `getFullname("", "");` `getFullname(true, true);`, what should be returned?
In the assignment was also mentioned `Remember to consider someone calling the function with an empty string as firstname and lastname.`

Reply:
> Both are const. Why let? Good reading https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/

yes, const
good article thanks

Reply:
> Try also `getFullname();` `getFullname("", "");` `getFullname(true, true);`, what should be returned? In the assignment was also mentioned `Remember to consider someone calling the function with an empty string as firstname and lastname.`

it should return error message in the cases above.